### DISCUSSION TITLE
nice-article/juejin/yigppttdcxyqddnzzj

### DISCUSSION THREAD

**Discussion by giscus** [itwanger/tobebetterjavaer-giscus#145] [Announcements]
# nice-article/juejin/yigppttdcxyqddnzzj

一个程序媛前端在南京漂流的6年。6年，弹指一挥间。 时间啊，不等人。这些年的心路历程。从实习找工作到毕业，到惶恐，到结婚，总结记录一下

https://tobebetterjavaer.com/nice-article/juejin/yigppttdcxyqddnzzj.html

**Comment by lyzerodragon** [comment]
羡慕捏，恐恋，恐婚

**Reply by lyzerodragon** [reply]
[老路小路](https://music.163.com/song?id=105886&userid=1895787014)
